### PR TITLE
Ticket creation form

### PR DESCRIPTION
**Pull request by benwerd**
# Description

This is the initial ticket creation form, which takes locks from redux and allows you to create an event based on them.

Uploading this as a draft PR because I can't for the life of me find a way to test `react-select` options. All ideas appreciated with thanks.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

Screen Shot 2019-04-15 at 4 57 14 PM
Screen Shot 2019-04-15 at 4 57 30 PM

### CODE REVIEW

**Comment by julien51** [comment]
> Uploading this as a draft PR because I can't for the life of me find a way to test react-select options. All ideas appreciated with thanks.

You need to mock it. Look at the stuff in the DatePicker

File: tickets/jest.config.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = {
   coverageThreshold: {
     global: {
       branches: 77.45,
-      functions: 89.55,
+      functions: 86.55,
```
Comment (julien51) [inline tickets/jest.config.js]: A bit worrisome. Any idea why this goes down so much?

File: tickets/src/components/content/CreateContent.js [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+import BrowserOnly from '../helpers/BrowserOnly'
+import { pageTitle } from '../../constants'
+
+export class CreateContent extends Component {
+  constructor(props) {
+    super(props)
+    const { now, locks } = props
+
+    this.state = {
+      lock: locks[0] || '',
```
Comment (julien51) [inline tickets/src/components/content/CreateContent.js]: We should probably show an error to the user when they do not have locks deployed?

File: tickets/jest.config.js (hunk 1 above)
Comment (benwerd) [reply tickets/jest.config.js]: The total was actually 88.49 - but I'm giving a bit of a margin for the next set of PRs. I need to find a better way to test the internal methods that are called by the react-select entity.

File: tickets/src/components/content/CreateContent.js (hunk 2 above)
Comment (benwerd) [reply tickets/src/components/content/CreateContent.js]: Or maybe just don't show the pulldown?

Comment (benwerd) [reply tickets/src/components/content/CreateContent.js]: We already have a link to create new locks

Comment (julien51) [reply tickets/src/components/content/CreateContent.js]: They can work for now

**Pull request merged**